src/style.css

```
@@ -0,0 +1,54 @@
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+
+  height: 96px;
+  width: 96px;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.stars__star {
+  content: "";
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4),
```

fix for every star like this

```suggestion
.stars--4 .stars__star:nth-child(-n+4),
```